code/game/objects/buckling.dm

```diff
@@ -141,7 +141,7 @@
 	if(isguardian(user) && (M.loc == user.loc || user.alpha == 60)) //Alpha is for detecting ranged guardians in scout mode
 		return  //unmanifested guardians shouldn't be able to buckle mobs
 
-	if(M != user && (!in_range(M, src) || !do_after(user, 1 SECONDS, target = M)))
+	if(M != user && !M.incapacitated() && (!in_range(M, src) || !do_after(user, 1 SECONDS, target = M)))
```

Comment:
we're using the HANDS_BLOCKED trait instead of incapacitated calls now, please switch it to that trait